code/modules/admin/species_ban.dm

```diff
@@ -0,0 +1,107 @@
+// Species ban system.
+// Disallows a player from using a specific species, but lets them play the game regardless
+// Useful if someone isnt playing a species properly at all (IE: Not following hard-driven species lore)
```

Comment:
We don't enforce Species Lore on Paradise.
If this is the example for why the feature is useful, its a bad example.

Reply:
If you can provide a better one, ill change it